ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/DiskPolicy.hs

```diff
@@ -17,6 +18,11 @@ import           Control.Monad.Class.MonadTime
 
 import           Ouroboros.Consensus.Config.SecurityParam
 
+
+data RequestedInterval =
```

Comment:
I think we should rename this to `RequestedDiskSnapshotInterval`. Especially when it occurs in eg `Node.hs` as `srnRequestedInterval`, "RequestedInterval" doesn't indicate the semantics as much as I'd like it too.

Also, some Haddock, especially indicating that the `Word64` is seconds.

Reply:
renamed to `RequestedSnapshotInterval`

Reply:
> Also, some Haddock explaining the semantics and motivation, especially indicating that the Word64 is seconds.

I shouldn't have packed this bit into the same comment ^^^